src/types/appData.ts

```diff
@@ -0,0 +1,4 @@
+export enum AppDataVisibility {
+  MEMBER = 'member',
+  ITEM = 'item',
+}
```

Comment:
Is this not defined inside @graasp/sdk ? Should we define it there, so you don't need to re-define it here ?

Reply:
No, this is not in the sdk. But we may add it. What do you think?